### PR TITLE
Use the @Prestashop notation

### PR DESCRIPTION
**Pull request by PrestaEdit**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 8.x
| Description?  | Does not found the pull request where it was introduced but you can now use the `@PrestaShop` notation.

### CODE REVIEW

**Review by PululuK — APPROVED** [review]

**Review by leemyongpakvn — COMMENTED** [review]
According to https://devdocs.prestashop.com/8/modules/core-updates/8.0/ you need to change the second colon as well, like this:
{% form_theme form '@PrestaShop/Admin/TwigTemplateForm```/```prestashop_ui_kit_base.html.twig' %}

**Comment by PrestaEdit** [comment]
Hi @leemyongpakvn,

Could you precise me what about your refer and what you want to choose here ?

File: development/components/form/form-theme/form-theme.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ This theme extends Symfony's Bootstrap 4 form theme, allowing it to inherit all
 Until it becomes the default Form Theme, Twig templates that need to rely on this theme need to activate it using the following statement:
 
 ```
-{% form_theme form 'PrestaShopBundle:Admin/TwigTemplateForm:prestashop_ui_kit_base.html.twig' %}
+{% form_theme form '@PrestaShop/Admin/TwigTemplateForm:prestashop_ui_kit_base.html.twig' %}
```
Comment (kpodemski) [inline development/components/form/form-theme/form-theme.md]:
```suggestion
{% form_theme form '@PrestaShop/Admin/TwigTemplateForm/prestashop_ui_kit_base.html.twig' %}
```

**Comment by PrestaEdit** [comment]
Thanks @kpodemski ; I see now what I forgot to change also there ! :)

**Review by kpodemski — APPROVED** [review]

**Pull request merged**